backend/routes/data.py

```diff
@@ -22,6 +23,54 @@ def send_audio_file(file_name):
     return send_from_directory(app.config["UPLOAD_FOLDER"], file_name)
 
 
+def validate_segmentation(segment):
```

Comment:
This validation is incorrect. We don't expect users to pass `created_at` and `last_modified` timestamps. Those are automatically filled when data is inserted.

Secondly, we wish to add datapoint only if all segmentations are valid. See the comments below regarding skipping segments.